README.md

```diff
@@ -490,6 +490,66 @@ async function run () {
 run()
 ```
 
+#### app.graphql.replaceSchema(schema)
+
+It is possible to replace schema and resolvers using `makeSchemaExecutable` function in separate fastify plugins, like so:
+
+```js
+const app = Fastify()
+
+app.register(GQL, {
+  schema: makeExecutableSchema({
```

Comment:
This is imported from somewhere, can you add the relevant requires at the top?

Reply:
I've added import for `makeExecutableSchema`